src/Schema/AbstractSchemaManager.php

```diff
@@ -309,7 +309,7 @@ protected function doListTableIndexes($table): array
      *
      * @throws Exception
      */
-    public function tablesExist($names)
+    public function tablesExist($names, $lowerCase = true)
```

Comment:
This doesn't look like the right solution. When it comes to object names, the DBAL should behave the same as the underlying database without the user having to provide any additional input.

Reply:
Re: "DBAL should behave the same as the underlying database":
DBAL will be the same as the underlying database without the user having to provide any additional input.
That's why the argument "$lowerCase" is added as an optional parameter: $lowerCase = true and can be ignored.

Means:

Can be used in two scenarios:

Scenario "A": $this->tablesExist([UserNames, Credentials], false);
Scenario "B": $this->tablesExist([UserNames, Credentials]);

"A" and "B" - both scenarios properly records. DBAL / Object names: safe. 

Can the function be implemented without optional argument "$lowerCase = true":
Correct: can be.

But, can you imagine how many projects can be
who used exactly this feature - naming tables in lower case.

The main goal of the optional argument: "$lowerCase = true" is:
give backwards compatibility for projects, who used exactly this feature - naming tables in lower case.

Thank you